docs/example-react-hooks-useContext.md

```diff
@@ -0,0 +1,94 @@
+This is a simple example to test a component using the useContext hook. 
+
+
+The context object
+
+```
+import React from 'react';
```

Comment:
Hi! Some parts of the code are using semicolons, and some others aren't. I just checked and I'd say all demo code from the docs are written without semicolons, so I'd suggest removing them from the examples too!

Reply:
If you pull down the repo locally the git hooks should format the docs on save